vent/controller/control_module.py

```diff
@@ -628,7 +720,7 @@ class ControlModuleDevice(ControlModuleBase):
     # Implement ControlModuleBase functions
     def __init__(self):
         ControlModuleBase.__init__(self)
-        self.HAL = io.Hal()
+        self.HAL = io.Hal(config_file='vent/io/config/dinky-devices.ini')
```

Comment:
do we want to keep this hardcoded?

Reply:
Would this go in values.py? Unclear to me where to safe the default path

Reply:
check out vent.common.prefs ;) 

I'll use that to collect prefs and manage persistent state

Reply:
check out vent.common.prefs ;) 

I'll use that to collect prefs and manage persistent state